### ISSUE TITLE
`-Wdocumentation` false positive for function declaration with varargs

### ISSUE THREAD

**Issue by firewave** [llvm/llvm-project#65296]
```cpp
/**
 * @param m, the loglevel
 */
void f(const char *m, ...);
```

```
<source>:2:11: warning: parameter 'm,' not found in the function declaration [-Wdocumentation]
    2 |  * @param m, the loglevel
      |           ^~
<source>:2:11: note: did you mean 'm'?
    2 |  * @param m, the loglevel
      |           ^~
      |           m
```
https://godbolt.org/z/fGKnhh68h

Related to #44510.
